test/e2e/egress_traffic_test.go

```diff
@@ -0,0 +1,66 @@
+package e2e
+
+import (
+	"encoding/base64"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/knative/serving/test"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	targetHostDomain = "httpbin.org"
```

Comment:
Lets do `www.google.com` here as:

1. Frequency - I don't want whoever runs httpbin.org to end up with a large bill for ingress traffic
2. Performance - google.com should resolve DNS and return quickly across multiple geographies
3. Consistency - reliability of httpbin.org is unknown and the google.com homepage should be fairly high

Reply:
SGTM